shm/inc/Player.hpp

```diff
@@ -1,25 +1,23 @@
 #pragma once
 
-#include "shm/inc/Cargo.hpp"
-#include "shm/inc/Ship.hpp"
+#include "shm/inc/Ship.hpp" // forward declaration not enough
 
-#include <algorithm>
-#include <memory>
-#include <numeric>
+class Cargo;
 
+// class responsible for manage user in game
 class Player {
 public:
-    Player(std::unique_ptr<Ship>, size_t, size_t);
+    Player(std::unique_ptr<Ship> ship, size_t money, size_t availableSpace);
 
     std::unique_ptr<Ship> getShip() { return std::move(ship_); };
```

Comment:
forward declaration not enough because of this. If you move it to .cpp class it should work